content/wasm-functions/using-cron-trigger.md

```diff
@@ -7,18 +7,20 @@ enable_shortcodes = true
 - [Prerequisites](#prerequisites)
 - [Introduction to Cron Jobs](#introduction-to-cron-jobs)
   - [What Are Cron Jobs?](#what-are-cron-jobs)
-- [Spin and Fermyon Wasm Functions](#spin-and-fermyon-wasm-functions)
-  - [Create Spin application](#create-spin-application)
-  - [Creating a Cron Job for your Spin application](#creating-a-cron-job-for-your-spin-application)
-  - [Managing Cron Jobs](#managing-cron-jobs)
+- [Create Spin application](#create-spin-application)
+  - [Deploy Spin Application to Fermyon Wasm Functions](#deploy-spin-application-to-fermyon-wasm-functions)
+- [Creating a Cron Job for Your Spin Application](#creating-a-cron-job-for-your-spin-application)
+- [Managing Cron Jobs](#managing-cron-jobs)
+- [Delete a Cron Job](#delete-cron-job)
+- [Next Steps](#next-steps)
   
-This tutorial guides you through the process of scheduling HTTP requests in a Spin application running on Fermyon Wasm Functions using cron jobs. The `spin aka cron` command allows applications to receive scheduled HTTP triggers without relying on external services. In this tutorial, you'll learn how to define and manage cron jobs using the `spin aka cron` command.
+This tutorial guides you through the process of scheduling HTTP requests in a Spin application running on Fermyon Wasm Functions using cron jobs. The `spin aka crons` command allows applications to receive scheduled HTTP triggers without relying on external services. In this tutorial, you'll learn how to define and manage cron jobs using the `spin aka crons` command.
```

Comment:
This page seems to be largely the same content as the previous one but with the "cron trigger" language back in... is this meant to have been deleted in favour of the page above?  The TOC entry still points to this one.